cli/batch/service.py

```diff
@@ -0,0 +1,34 @@
+"""
+Provide implementation of the batch service.
+"""
+from accessify import implements
+
+from cli.batch.interfaces import BatchInterface
+
+
+@implements(BatchInterface)
+class Batch:
+    """
+    Implements batch interface.
+    """
+
+    def __init__(self, service):
+        """
+        Constructor.
+
+        Arguments:
+            service: object to interact with Remme core API.
+        """
+        self.service = service
+
+    async def get_list(self, query=None):
+        """
+        Get all batches from REMChain.
+
+        Arguments:
+            query (dict, optional): dictionary with specific parameters
+
+        Returns:
+            List of batches.
+        """
+        return await self.service.blockchain_info.get_batches(query=query)
```

Comment:
Actualize the service as here — https://github.com/Remmeauth/remme-core-cli/blob/develop/cli/account/service.py#L28

Reply:
Done.